src/axom/inlet/tests/inlet_object.cpp

```diff
@@ -780,6 +812,29 @@ TYPED_TEST(inlet_object, nested_array_of_nested_structs)
   EXPECT_EQ(quuxs_with_foo, expected_quuxs);
 }
 
+TYPED_TEST(inlet_object, nested_array_of_struct_verify_pass)
+{
+  std::string testString = "quux = { }";
+  DataStore ds;
+  Inlet inlet = createBasicInlet<TypeParam>(&ds, testString);
+
+  auto& quux_schema = inlet.addStructArray("quux");
+
+  // Simple required field
+  quux_schema.addString("corge").required();
```

Comment:
`"quux"` is an array of structs. If it had a any elements, each would require a `"corge"` member. Since the array is empty, it's fine.

I think this is the kind of confusion that stems from `addStructArray()` returning the definition to the items of the array, rather than the array itself. There's no way to distinguish between the two. This is outside the scope of this PR, though.